docs/transpile/qiskit-transpiler-service.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Transpile circuits remotely with the Qiskit transpiler service
+description: What is the Qiskit transpiler service and how to use it
+
+---
+# Transpile circuits remotely with the Qiskit transpiler service
+
+The Qiskit transpiler service provides transpilation capabilities on the cloud.
+In addition to the traditional [Qiskit transpiling capabilities](index), the Qiskit transpiler service allows users to take advantage of IBM Quantum Cloud resources to run their transpiling tasks and use the latest transpiling technology enhanced by the AI-powered transpiler passes.
+
+The Qiskit transpiler service offers a Python library to seamlessly integrate this service and its capabilities into your current Qiskit patterns and workflows.
+
+<Admonition type="note">
+    This experimental service is only available for IBM Quantum Premium Plan users.
+    The service is an alpha release, subject to change.
+</Admonition>
+
+
+## Install the qiskit-transpiler-service package
+
+To use the Qiskit transpiler service, install the `qiskit-transpiler-service` package:
+
+```sh
+pip install qiskit-transpiler-service
+```
+
+By default, the package will try to authenticate to IBM Quantum services with the defined Qiskit API token, and will take your token from the `QISKIT_IBM_TOKEN` environment variable or from the file `~/.qiskit/qiskit-ibm.json` (under the section `default-ibm-quantum`).
+
+### qiskit-transpiler-service transpile options
```

Comment:
I applaud the use of the "twistie" feature :-) but since the section within the twistie isn't very long, I've removed it so that the material is more easily accessible. Feel free to reinstate the twistie if you have strong feelings about it!

Reply:
I'm ok! Thanks